src/main/java/com/minecolonies/items/ItemScepterGuard.java

```diff
@@ -0,0 +1,133 @@
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Guard Scepter Item class. Used to give tasks to guards.
+ */
+public class ItemScepterGuard extends AbstractItemMinecolonies
+{
+    /**
+     * The compound tag for the last pos the tool has been clicked.
+     */
+    private static final String TAG_LAST_POS = "lastPos";
+
+    /**
+     * Caliper constructor. Sets max stack to 1, like other tools.
+     */
+    public ItemScepterGuard()
+    {
+        super("scepterGuard");
+        this.setMaxDamage(2);
+
+        maxStackSize = 1;
+    }
+
+    @NotNull
+    @Override
+    public EnumActionResult onItemUse(
+                                       final ItemStack scepter,
+                                       final EntityPlayer playerIn,
+                                       final World worldIn,
+                                       final BlockPos pos,
+                                       final EnumHand hand,
+                                       final EnumFacing facing,
+                                       final float hitX,
+                                       final float hitY,
+                                       final float hitZ)
+    {
+        //todo watch how interaction with server is, might facilitate this.
+        // if server world, do nothing
+        if (worldIn.isRemote)
+        {
+            return EnumActionResult.FAIL;
+        }
+
+        if (!scepter.hasTagCompound())
+        {
+            scepter.setTagCompound(new NBTTagCompound());
+        }
+        final NBTTagCompound compound = scepter.getTagCompound();
+
+        if (compound.hasKey(TAG_LAST_POS))
+        {
+            final BlockPos lastPos = BlockPosUtil.readFromNBT(compound, TAG_LAST_POS);
+            if (lastPos.equals(pos))
+            {
+                playerIn.inventory.removeStackFromSlot(playerIn.inventory.currentItem);
+                LanguageHandler.sendPlayerMessage(playerIn, LanguageHandler.format("com.minecolonies.job.guard.toolDoubleClick"));
+                return EnumActionResult.FAIL;
+            }
+        }
+        return handleItemUsage(worldIn, pos, compound, playerIn);
+    }
+
+    /**
+     * Handles the usage of the item.
+     *
+     * @param worldIn  the world it is used in.
+     * @param pos      the position.
+     * @param compound the compound.
+     * @param playerIn the player using it.
+     * @return if it has been successful.
+     */
+    @NotNull
+    private static EnumActionResult handleItemUsage(final World worldIn, final BlockPos pos, final NBTTagCompound compound, final EntityPlayer playerIn)
+    {
+        final Colony colony = ColonyManager.getClosestColony(worldIn, pos);
+        if (colony == null)
+        {
+            return EnumActionResult.FAIL;
+        }
+
+        final BlockPos guardTower = BlockPosUtil.readFromNBT(compound, "pos");
+        final AbstractBuilding hut = colony.getBuilding(guardTower);
+        if (hut == null || !(hut instanceof BuildingGuardTower))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) No need to check for null before an instanceof [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ASimplifyConditional)